Sources/Modules/Food/Projectiles/CreamChocolateCupStaff_proj.cs

```diff
@@ -0,0 +1,252 @@
+using Everglow.Food.Items.Weapons;
+using Terraria.DataStructures;
+
+namespace Everglow.Food.Projectiles;
+public class CreamChocolateCupStaff_proj : ModProjectile
+{
+	public override void SetDefaults()
+	{
+		Projectile.width = 36;
+		Projectile.height = 36;
+		Projectile.aiStyle = -1;
+		Projectile.timeLeft = 360000;
+		Projectile.tileCollide = false;
+		Projectile.DamageType = DamageClass.Magic;
+	}
+	public float WeaponShake = 0;
+
+	public override void OnSpawn(IEntitySource source)
+	{
+		Player player = Main.player[Projectile.owner];
+		Vector2 mouseToPlayer = Main.MouseWorld - player.MountedCenter;
+		mouseToPlayer = Vector2.Normalize(mouseToPlayer);
+		Projectile.NewProjectileDirect(Projectile.GetSource_FromAI(), Projectile.Center, mouseToPlayer * 16f, ModContent.ProjectileType<CreamChocolateCup_ChocolateBars>(), Projectile.damage * 3, 0.4f, player.whoAmI);
```

Comment:
The chocolate projectiles deal 3x base damage, which makes it deal around 100 damage which can already take the player to hardmode (unless it is a hardmode weapon). 2x base damage can be done instead.
```suggestion
		Projectile.NewProjectileDirect(Projectile.GetSource_FromAI(), Projectile.Center, mouseToPlayer * 16f, ModContent.ProjectileType<CreamChocolateCup_ChocolateBars>(), Projectile.damage * 2, 0.4f, player.whoAmI);
```
Also make the chocolate projectiles affected by gravity, and make cream projectiles appear. It gives the staff more flavour and uniqueness.
And sounds should also be added

Reply:
Right click to make cream projectile

Reply:
> Right click to make cream projectile

I see it now. What can happen on left click is that a small cream projectile can also appear with the chocolate projectile. It will look like `CreamChocolateCupStaff_cream.png`. This is up to you though